Boy or Girl/Boy or Girl/Program.cs

```diff
@@ -0,0 +1,51 @@
+﻿namespace Boy_or_Girl
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Reader reader = new Reader();
+            var userName = reader.Next();
+            var check = checkGirl(userName);
+            if (check == true)
+            {
+                Console.WriteLine("CHAT WITH HER!");
+            }
+            else
+            {
+                Console.WriteLine("IGNORE HIM!");
+            }
+        }
+        public static Boolean checkGirl(String userName)
+        {
+            if ((userName.Distinct().Count() % 2) != 0)
+            {
+                return false;
+            }
+            return true;
```

Comment:
We can shorten this function as following suggestion: 
```suggestion
            return !((userName.Distinct().Count() % 2) != 0);
```